qutebrowser/mainwindow/statusbar/url.py

```diff
@@ -28,8 +28,8 @@
 
 
 # Note this has entries for success/error/warn from widgets.webview:LoadStatus
-UrlType = usertypes.enum('UrlType', ['success', 'error', 'warn', 'hover',
-                                     'normal'])
+UrlType = usertypes.enum('UrlType', ['success', 'success_https', ' error',
```

Comment:
That space shouldn't be here :wink:

This lead to an exception whenever a page with an error was loaded, e.g. an inexistent host - I just fixed that :smile: